.travis.yml

```diff
@@ -0,0 +1,33 @@
+language: cpp
+python:
+  - "2.7"
+install:
+  # Install ARM GCC
+  - sudo add-apt-repository ppa:team-gcc-arm-embedded/ppa -y
+  - sudo apt-get update
+  - sudo apt-get install gcc-arm-embedded -y
+  # Install yotta
+  - pip install --user yotta
+script:
+  # Build all tests for frdm-k64f-gcc
+  - yotta target frdm-k64f-gcc
```

Comment:
Good point, that could be added as well. If we can decide on a list of platforms to build I can add that to the PR.

Maybe we should build all boards on mbed.com that don't have the experimental support status? Right now that's K64F and the mkit. Any objections?